src/main/java/mServer/crawler/sender/phoenix/tasks/PhoenixFilmDetailTask.java

```diff
@@ -0,0 +1,151 @@
+package mServer.crawler.sender.phoenix.tasks;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z_]+(\.[a-z_][a-z0-9_]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)